x-pack/plugins/security_solution/public/risk_score/containers/all/index.test.tsx

```diff
@@ -21,38 +22,56 @@ jest.mock('../../../common/hooks/use_space_id', () => ({
 }));
 
 jest.mock('../../../common/hooks/use_app_toasts');
+jest.mock('../feature_status');
 
-const mockUseMlCapabilities = jest.fn();
-jest.mock('../../../common/components/ml/hooks/use_ml_capabilities', () => ({
-  useMlCapabilities: () => mockUseMlCapabilities(),
-}));
-
+const mockUseRiskScoreFeatureStatus = useRiskScoreFeatureStatus as jest.Mock;
 const mockUseSearchStrategy = useSearchStrategy as jest.Mock;
 const mockSearch = jest.fn();
-const mockRefetch = jest.fn();
 
 let appToastsMock: jest.Mocked<ReturnType<typeof useAppToastsMock.create>>;
 
 [useHostRiskScore, useUserRiskScore].forEach((fn) => {
   const riskEntity = fn.name === 'useHostRiskScore' ? 'host' : 'user';
+  const defaultFeatureStatus = {
```

Comment:
@angorayc sorry you might get merge conflicts for changes in these tests. I moved all the props/args/results into `const`s so that it will be more readable below what is being tested.